tests/tpch/conftest.py

```diff
@@ -44,10 +58,13 @@ def dataset_path(local, scale):
     remote_paths = {
         10: "s3://coiled-runtime-ci/tpch/scale-10/",
         100: "s3://coiled-runtime-ci/tpch/scale-100/",
+        # 1000: "s3://coiled-runtime-ci/tpc-h/scale-1000/",
         1000: "s3://coiled-runtime-ci/tpch/scale-1000/",
-        10000: "s3://coiled-runtime-ci/tpch/scale-10000/",
+        10000: "s3://coiled-runtime-ci/tpc-h/scale-10000/",
+        # 10000: "s3://coiled-runtime-ci/tpch/scale-10000/",
     }
```

Comment:
This can now be:

```python
    remote_paths = {
        10: "s3://coiled-runtime-ci/tpc-h/scale-10/",
        100: "s3://coiled-runtime-ci/tpc-h/scale-100/",
        1000: "s3://coiled-runtime-ci/tpc-h/scale-1000/",
        10000: "s3://coiled-runtime-ci/tpc-h/scale-10000/",
    }
```
I'm generating the "strict" schema versions now (no casting of date or decimal types), maybe later you'll be interested in adding those as alternative options to run on.

Reply:
Thanks!